system/jlib/jsocket.cpp

```diff
@@ -4340,7 +4340,8 @@ class CSocketEpollThread: public CSocketBaseThread
             else
                 dummysock = ::socket(AF_INET, SOCK_STREAM, 0);
             CHECKSOCKRANGE(dummysock);
-            epoll_op(epfd, EPOLL_CTL_ADD, dummysock, (EPOLLIN | EPOLLERR));
+            // added EPOLLIN also because cannot find anywhere OOB is sent
```

Comment:
I had to look up what OOB was (MSG_OOB), would be worth expanding comment a bit.